Procfile

```diff
@@ -1,2 +1,4 @@
 web:       gunicorn cabot.wsgi:application --config gunicorn.conf
 celery:    celery worker -B -A cabot --loglevel=INFO --concurrency=16 -Ofair
+celerybeat: python manage.py celery beat
```

Comment:
If there's a separate celerybeat worker we should remove the `-B` option from the celery process